Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM bitriseio/docker-bitrise-base:latest
```

Comment:
Building a custom image is not necessary, as you can run it interactively in the container while preserving current state of source files:
```sh
docker run -v $(pwd):/bitrise/src bitriseio/docker-bitrise-base:latest /bin/bash -c "bitrise update && bitrise run test"
```

I suggest replacing this file with the script above, called `test-linux.sh`

Reply:
Done, I've removed the Dockerfile and replaced it with provided bash script

Reply:
cool 👍